src/sensor.py

```diff
@@ -0,0 +1,10 @@
+class Sensor:
+
+    def __init__(self, name):
+        self.name = name
+
+    def measure(self):
```

Comment:
The `measure` and `render` methods should be overridden by any concrete implementation to start a measurement and to provide a HTML snipped that renders the interface.